backend/Squirrel.ConsoleApp/Extensions/HubConnectionExtensions.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Microsoft.AspNetCore.SignalR.Client;
+using Squirrel.ConsoleApp.BL.Interfaces;
+using Squirrel.ConsoleApp.Models;
+using Squirrel.ConsoleApp.Services;
+using System.Threading.Channels;
+
+namespace Squirrel.Core.WebAPI.Extensions;
+
+public static class HubConnectionExtensions
+{
+    private static bool CurrentClientIsTarget(string currentClientId, string clientId)
+    {
+        if (string.IsNullOrEmpty(currentClientId))
+        {
+            return false;
+        }
+
+        if (!clientId.Equals(currentClientId))
+        {
+            return false;
+        }
+
+        return true;
+    }
+
+    public static void RegisterActions(this HubConnection hubConnection, string currentClientId, IGetActionsService getActionsService)
+    {
+        hubConnection.On("GetAllTablesNamesAsync", (string clientId) =>
```

Comment:
let's create some Enun for these invoke names (like `GetAllTablesNamesAsync`, `ReceiveAllTablesNamesAsync` etc) not just simple strings

Reply:
This will not save us from the main problem - the mapping of server-side method names and client-side registered names. These "method name strings" are all in one place and it is easy to check or change them. If we have an additional Enum - we will have to change the same in different places. 
You know that we have an interface which describe tha same name methods. I guess it enough, but I will ask about that tomorrow.

Reply:
Decided use strings